modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/AbstractPageMemoryTableStorage.java

```diff
@@ -103,7 +105,7 @@ public void stop() throws StorageException {
         busyLock.block();
 
         try {
-            IgniteUtils.closeAll(mvPartitionStorages.getAllForClose().stream().map(mvPartitionStorage -> mvPartitionStorage::close));
+            IgniteUtils.closeAllManually(mvPartitionStorages.getAllForCloseOrDestroy().get(10, TimeUnit.SECONDS).stream());
```

Comment:
Please add a comment about 10 seconds, explaining why it's enough.

Reply:
Maybe extract a future into a variable, so the code will not look as tight

Reply:
Try to fix it.